sklearn/model_selection/_validation.py

```diff
@@ -592,14 +598,8 @@ def _score(estimator, X_test, y_test, scorer, is_multimetric=False):
             score = scorer(estimator, X_test)
         else:
             score = scorer(estimator, X_test, y_test)
-
         if hasattr(score, 'item'):
-            try:
-                # e.g. unwrap memmapped scalars
-                score = score.item()
-            except ValueError:
```

Comment:
I don't get what this has to do with the present pull request

Reply:
I was trying to improve test coverage. Not sure in what situation the item() method can fail (we're checking before that if the object has this method before calling it), that's why I removed that try-except. But can revert that change if you feel was correct (and add a test if you give me an example of how to make it fail)

Reply:
I don't know why it's useful really, and I don't know why it's not tested, but you can't just remove existing functionality...

Reply:
yeah sorry

> @marctorrellas if you need help with test tell me I can help =)

@eamanu I would actually appreciate if you can revert my change and add a test for that .item() failure. Or just giving me an example of how to make that to fail in purpose. I mean, can do it totally artificially, but there's a comment there "unwrap memapped scalars" and probably the tests should use that but no clue what it is

Reply:
Blame shows this commit:
https://github.com/amueller/scikit-learn/commit/a08555a2384884c03d5deb509192a052c06caa85
which is not very helpful.
I had assumed this was for dask support, but it would be weird if it was introduced in that PR.
If neither @jnothman nor I know why it's there it's a bit suspect but I wouldn't worry about it for this PR.

Reply:
(i.e. undo the change)

Reply:
done